static/src/images/global/information-circle-grey.svg

```diff
@@ -0,0 +1 @@
+<svg width="14" height="14" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 14 14"><path fill="#767676" d="M7 0C3.1 0 0 3.1 0 7s3.1 7 7 7 7-3.1 7-7-3.1-7-7-7zm.9 10l-.3 1c-.4.2-1.2.4-1.9.4-.5 0-.8-.1-.8-.6 0-.1 0-.3.1-.4L6.3 6h-.9l.3-.8 2.7-.3.2-.1L6.8 10h1.1zm-.2-5.8c-.6 0-.9-.3-.9-.8 0-.6.5-1 1.1-1 .6 0 .9.3.9.8 0 .6-.5 1-1.1 1z"/></svg>
```

Comment:
Hey I may be wrong. But I had to leave it in here, cos this svg gets base 64 encoded and inserted as a pseudo element. PLZ HELP!